scripts/globals/abilities/spirit_link.lua

```diff
@@ -69,9 +85,42 @@ function onUseAbility(player,target,ability)
     if (pet:getHP() < pet:getMaxHP()) then -- sleep is only removed if it heals the wyvern
         removeSleepEffects(pet)
     end
-
+    
+    -- Empathy copying
+    local empathyTotal = player:getMerit(dsp.merit.EMPATHY)/5 -- +5 value per increase for this merit... not sure why
```

Comment:
In the merits.sql table, change the "value" entry for Empathy from 5 to 1 so you don't have to do the /5.

Reply:
> In the merits.sql table, change the "value" entry for Empathy from 5 to 1 so you don't have to do the /5.

updated, thanks